src/main/views/macro/externalLink.njk

```diff
@@ -1,2 +1,4 @@
-{% macro externalLink(text, href) %}<a href="{{ href }}" target="_blank">{{ text }}</a>
-  <span class="font-xsmall">{{ t('(opens in a new window)') }}</span>{% endmacro %}
+{% macro externalLink(text, href) %}
+  <a href="{{ href }}" target="_blank">{{ text }}</a>
```

Comment:
Can you check that moving this down to  a new line hasn't caused any spacing issued please? i.e. there isn't any spaces now where there shouldn't be

Reply:
This is how it looks - 
<img width="436" alt="screen shot 2018-04-19 at 08 45 34" src="https://user-images.githubusercontent.com/12914259/38978033-11a44812-43ae-11e8-93ea-1df55ce2a4f7.png">